kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -116,6 +116,13 @@
       this.initializeQuiz(this.$route.params.classId);
       this.quizInitialized = true;
     },
+    methods: {
+      saveQuizAndRedirect() {
+        this.saveQuiz().then(() => {
+          this.$router.replace({ name: PageNames.EXAMS });
```

Comment:
This route requires a classId parameter: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/routes/planExamRoutes.js#L31